app/admin/widgets/filter/scope_selectlist.blade.php

```diff
@@ -0,0 +1,26 @@
+@php
+    $isCheckboxMode = $scope->mode ?? 'checkbox';
+    $selectMultiple = $isCheckboxMode == 'checkbox';
+    $options = $this->getSelectListOptions($scope->scopeName);
```

Comment:
I don't think we need the new method here. Use `getSelectOptions` here and cast `$options['active']` to array.

Reply:
Ok changed